<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>769</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>yeoncho reading word</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Zoom.svg</normaloff>Zoom.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QTextEdit" name="te">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>490</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans KR</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="word_mean_ComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>491</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="word_next_PushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>720</y>
      <width>88</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="word_rm_PushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>680</y>
      <width>88</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="word_name_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_check">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>50</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="book_page_SpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="word_mean_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>491</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="word_line_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>a:제목
b:단어풀이
c:읽기 전에
d:도움말
e:페이지 옆</string>
    </property>
   </widget>
   <widget class="QLabel" name="persent_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
